contracts/src/Utilities/README.md

```diff
@@ -0,0 +1,18 @@
+## Utility Contracts
```

Comment:
nit: thoughts on changing directoyr name `Utilities` -> `utils` or `utilities`? My pref is `utils` but might look off for the abi-binding generated file

Reply:
Right now I prefer `Utilities` to match `Mocks` and `Teleporter` directories as we have them now. We do have #410 issue open and if we want to lower-case all of them we can do it there?

Reply:
sgtm